app/models/interest.rb

```diff
@@ -2,4 +2,7 @@ class Interest < ApplicationRecord
   self.inheritance_column = :_type_disabled
   enum type: { health: 0, hobby: 1, work: 2 }
   belongs_to :user
+  scope :amount_interest_cosmhealth_young_woman, -> { where("type = 0 and name like 'cosm%'").
```

Comment:
And i have on anyother idea what could be do from you suggested but name i real hard to named 
i push new commit and wait for your feedback.